debugger/pythonExtension/gdb_sgx_plugin.py

```diff
@@ -90,6 +106,10 @@ def write_to_memory(addr, buf):
         print ("Can't access memory at {0:x}.".format(int(addr)))
         return -1
 
+def read_int_from_memory(addr, size):
```

Comment:
We should check whether `size` is in a valid set, i.e., {1, 2, 4, 8}?

Reply:
Thanks for the suggestion. Instead of testing integer sizes passed to read_int_from_memory, I have locked down the field sizes used in the contract. That is a more rigorous test and encompasses all the fields used in the contract.